packages/visualizations/src/components/Table/Row.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+    import Cell from './Cell';
+    import ZoomIcon from './ZoomIcon.svelte';
+    import type { Column, Rows } from './types';
+
+    export let columns: Column[];
+    export let rows: Rows | undefined;
+    export let record: Record<string, unknown>;
+
+    let isRowHovered = false;
+    let rowElement: HTMLTableRowElement;
+    $: onMousenter = () => {
+        if (rows?.onMouseEnter) {
+            rows.onMouseEnter(record);
+        }
+        isRowHovered = true;
+    };
+    $: onMouseleave = () => {
+        if (rows?.onMouseLeave) {
+            rows.onMouseLeave(record);
+        }
+        isRowHovered = false;
+    };
+
+    $: if (rowElement) {
+        if (rows) {
```

Comment:
IMO, the `rows` naming find its limit in this file. It's weird to have a plural form of the word in the `Row` component.

We are starting to have a lot of interactions for this component (sort and now row events). Maybe we could starting thinking of other system to propagate events